pkg/handler/enqueue_annotation.go

```diff
@@ -0,0 +1,101 @@
+package handler
```

Comment:
I had to add the header to the new file I added in my PR. I think you might hit that as well.

Reply:
```Checking for license header...
license header checking failed:
  /home/travis/gopath/src/github.com/operator-framework/operator-sdk/pkg/handler/enqueue_annotation.go
make: *** [test/sanity] Error 255``